zerberus/zerberus.h

```diff
@@ -97,6 +98,8 @@ class Zerberus : public Ms::Synthesizer {
       Channel* channel(int n)       { return _channel[n]; }
       int loadProgress()            { return _loadProgress; }
       void setLoadProgress(int val) { _loadProgress = val; }
+      bool loadWasCanceled()        { return _loadWasCanceled; }
+      void setLoadWasCanceled(bool status)     { _loadWasCanceled = status; }
```

Comment:
This is the best wording for these function names that I could think of that still fit with the loadPogress() and setLoadProgress()